cocos/platform/desktop/CCGLViewImpl-desktop.h

```diff
@@ -79,6 +79,14 @@ class CC_DLL GLViewImpl : public GLView
     void pollEvents() override;
     GLFWwindow* getWindow() const { return _mainWindow; }
 
+    bool isFullscreen();
+    void setFullscreen();
+    void setFullscreen(int monitorIndex);
+    void setFullscreen(const GLFWvidmode &videoMode, GLFWmonitor *monitor);
```

Comment:
The method `setFullscreen(int monitorIndex)` lets the game dev add options to switch the game to another monitor, if more than one is present. I personally use it along with `getMonitorCount()` for this.

The other method `setFullscreen(const GLFWvidmode &videoMode, GLFWmonitor *monitor);` is present to give game devs the parallel of `createWithFullScreen(const std::string& viewName, const GLFWvidmode &videoMode, GLFWmonitor *monitor)` which some might be using when creating the window.